fiftyone/utils/cvat.py

```diff
@@ -4290,8 +4293,18 @@ def upload_samples(self, samples, backend):
                     project_id = self.create_project(project_name, cvat_schema)
                     project_ids.append(project_id)
 
-                _dataset_name = samples_batch._dataset.name.replace(" ", "_")
-                task_name = "FiftyOne_%s" % _dataset_name
+                # use custom task name if provided, else use default name
+                if config.task_name is None:
+                    _dataset_name = samples_batch._dataset.name.replace(
+                        " ", "_"
+                    )
+                    latest_anno_key = _get_latest_anno_key(samples)
```

Comment:
This is a clever workaround to get the `anno_key`, but now that `config.task_name` is supported, I think it would be best to just rely on the user putting their `anno_key` in the task name if that's what they want.

Reply:
It is unfortunate that we didn't include `anno_key` in either the `upload_annotations()` method or in the `AnnotationBackendConfig`, because it's definitely reasonable to want to use it to generate default values like this 🤦

Reply:
I guess it's enough for me that I can provide a custom task name to differentiate between tasks for the same dataset. 

Do you think that will require a lot of/breaking changes to pass `anno_key` to `upload_annotations`?